_posts/2016-04-19-shoutouts-week13.md

```diff
@@ -0,0 +1,65 @@
+---
+
+title: 'Shoutouts Week 13'
+author: Samantha 
+layout: blog-post
+comments: true
+
+---
+
+# Week 13 Shoutouts 
+
+Shoutouts has been away for longer than planned, but now we're back. To make it worth the wait, we've got even more great shoutouts than usual!
+
+We're lucky to have another brand new crop of first PRs, so to you all - welcome to Hoodie!
+
+First PRs:
+
+* [Alecs Konson](https://github.com/konson)
+* [Caity](https://github.com/caityc)
+* [Chris](https://github.com/chrifpa)
+* [Cody](https://github.com/codyzu)
+* [David Hope](https://github.com/HopeMobile)
+* [hintsa](https://github.com/hintsa)
+* [Krista Prenesti](https://github.com/kprenesti)
+* [Owen Williams](https://github.com/hyperperforator)
+* [Raquel Moss](https://github.com/raquelxmoss)
+* [Semere Lemma](https://github.com/gitsemere)
+
+Next, a big thanks to those who've chimed in with comments on various bits and bugs of Hoodie:
+
+Commenters:
+
+* [KaLaUnlimited](https://github.com/KaLaUnlimited)
+* [Annika](https://github.com/NeekyRabbit)
+* [Antoine](https://github.com/antoinelyset)
+* [Bryan](https://github.com/bryanbierce)
+* [Chris](https://github.com/chrisdemars)
+* [Charles](https://github.com/doublerebel)
+* [Henry](https://github.com/hzoo)
+* [Jules](https://github.com/jo-porter)
+* [Mattie](https://github.com/matsieftw)
+* [Nolan](https://github.com/nolanlawson)
+* [Alex](https://github.com/espy)
+* [Pawan](https://github.com/pawan92)
+* [Stephanie](https://github.com/rubymorillo)
+* [Sergi](https://github.com/sergi)
+* [shyNotSly](https://github.com/shyNotSly)
+* [Stephen](https://github.com/stevemasta34)
+* [Summer](https://github.com/summerspirit)
+* [Texassiga](https://github.com/texassiga)
+* [Kim](https://github.com/kmcrayton7)
```

Comment:
I’d suggest we leave out the commenters this time, and for next time we try to prepare better data to review the comments? Would that be okay with every one?

Besides that, it’s great and good to merge from my side